roles/api-environment/tasks/express-api.yml

```diff
@@ -24,3 +40,14 @@
       insertafter: "{{ disable_requiretty_for_api_user }}"
       line: "{{ api_user }} ALL=(ALL) NOPASSWD: {{ start_express_container_path }}"
     when: content_has_line is changed
+
+  - name: "build docker run image"
+    docker_image:
+      source: build
+      name: express-api-run
+      force: yes
```

Comment:
I got this deprecation warning: `[WARNING]: The "force" option will be removed in Ansible 2.12. Please use the "force_source", "force_absent" or "force_tag" option instead, depending on what you want to force.` Could you address this?